app/actions/definitions/documents.tsx

```diff
@@ -352,7 +352,10 @@ export const duplicateDocument = createAction({
  * of the collection for all collection members to see.
  */
 export const pinDocumentToCollection = createAction({
-  name: ({ t }) => t("Pin to collection"),
+  name: ({ t, stores }) => {
+    const activeCollectionName = stores.collections.active?.name;
```

Comment:
The active collection is not the collection it'll be pinned to, it's the collection associated with the `activeDocument` – they may be separate things